econplayground/templates/main/graph_confirm_delete.html

```diff
@@ -1,13 +1,26 @@
 {% extends 'base.html' %}
+{% load static econ_auth %}
```

Comment:
It doesn't look like you're using any `static` or `econ_auth` template tags in this template - this line can be removed.